app/controllers/user_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UserController < ApplicationController
+  def index
+  end
+  def show; end
+end
```

Comment:
- [ ] You have done a great job properly creating a Users controller, but you also have to create posts_controller with #index and #show actions for this milestone.
Kindly add posts_controller with #index and #show actions (same as users_controller)

Reply:
Hi @ShoiraTa 
I have created post controller

Reply:
Implemented 👍